tests/fixture_tailwind_remote_classes/nestedPlugins/nested/nested/nested/veryNestedPlugin.ts

```diff
@@ -0,0 +1,13 @@
+import { upNLevels } from "$fresh/plugins/tailwind.ts";
+import type { Plugin } from "../../../../../../src/server/types.ts";
+import VeryNestedPluginComponent from "../../../../components/VeryNestedPluginComponent.tsx";
+
+export const veryNestedPlugin = {
+  name: "very nested plugin",
+  location: import.meta.url,
+  projectLocation: upNLevels(import.meta.url, 5),
```

Comment:
Try using [`path.normazlie()`](https://deno.land/std@0.214.0/path/mod.ts?s=normalize) from `deno_std` instead.

Reply:
How will this help with a URL? e.g. `https://raw.githubusercontent.com/denoland/fresh/727d0e729e154323b99b319c04f9805f382949f0/tests/fixture_tailwind_remote_classes/hackyRemotePlugin.tsx` I guess I'm unclear what `normalize` would help with.

Reply:
Actually, check out [`url.normalize()`](https://deno.land/std/url/normalize.ts?s=normalize). That might help.

Reply:
I can of course change it to `projectLocation: normalize(import.meta.url + "../../../../../../").href` if you think that's better, but it seems really awkward to use.

Reply:
Yeah, I do think that's better 🙂

Reply:
Ok, I've removed the offending code. Note: in order to go up N levels, the user must now include N+1 copies of `../`.  This seems confusing, but maybe I'm just inexperienced with URL path modification.